README.md

```diff
@@ -468,3 +470,667 @@ Response:
     }
 }
 ```
+
+10. Filtering by attributes:
+
+You need use attributes query parameter which is an associative array where key is the attribute name and value is an array of attribute values.
+For e.g:
+```php
+$this->client->request('GET', '/shop-api/products', ['attributes' => ['Mug material' => ['Wood']]], [], ['ACCEPT' => 'application/json']);
```

Comment:
I don't have any idea, why the heck did I assume that it is a code. Fine then. Sorry, for false alarm :)